README.md

```diff
@@ -1,12 +1,144 @@
 # Oasis Metadata Registry Tools
 
-## Usage Examples
+[![CI test status][github-ci-tests-badge]][github-ci-tests-link]
+[![CI lint status][github-ci-lint-badge]][github-ci-lint-link]
 
-For some examples of using the Go library check the [`examples/`] directory. You
-can build all examples by running:
+<!-- markdownlint-disable line-length -->
+[github-ci-tests-badge]: https://github.com/oasisprotocol/oasis-core-rosetta-gateway/workflows/ci-tests/badge.svg
+[github-ci-tests-link]: https://github.com/oasisprotocol/oasis-core-rosetta-gateway/actions?query=workflow:ci-tests+branch:master
+[github-ci-lint-badge]: https://github.com/oasisprotocol/oasis-core-rosetta-gateway/workflows/ci-lint/badge.svg
+[github-ci-lint-link]: https://github.com/oasisprotocol/oasis-core-rosetta-gateway/actions?query=workflow:ci-lint+branch:master
+<!-- markdownlint-enable line-length -->
+
+This repository contains tools for working with the [Oasis Metadata Registry].
+
+[Oasis Metadata Registry]: https://github.com/oasisprotocol/metadata-registry
+
+## Building
+
+To build the `oasis-registry` tool, run:
+
+```sh
+make build
+```
+
+## Usage
+
+_NOTE: Currently, you will need to build the `oasis-registry` tool yourself._
+
+To sign an entity metadata statement, e.g.
+
+```json
+{
+  "v": 1,
+  "serial": 1,
+  "name": "My entity name",
+  "url": "https://my.entity/url",
+  "email": "my@entity.org",
+  "keybase": "my_keybase_handle",
+  "twitter": "my_twitter_handle"
+}
+```
+
+save it as a JSON file, e.g. `entity-metadata.json`, and run:
+
+```sh
+./oasis-registry/oasis-registry entity update \
+  <SIGNER-FLAGS> \
+  entity-metadata.json
+```
+
+where `<SIGNER-FLAGS>` are replaced by the appropriate signer CLI flags for your
+signer (e.g. Ledger-based signer, File-based signer).
```

Comment:
should there be a note here that ledger doesn't yet support singing metadata-registry statements

Reply:
:slightly_smiling_face: Yeah, good point. I'll add that.

Reply:
Added.